yellowbrick/features/pca.py

```diff
@@ -98,6 +102,8 @@ def __init__(self,
                  proj_dim=2,
                  proj_features=False,
                  color=None,
+                 legend_labels=None,
+                 label=None,
```

Comment:
I looked, but I can't see where this `label` param gets used -- can we remove this or did I miss something?

Reply:
you are correct @rebeccabilbro - I don't see it being used and it is not defined in the parameter list.  Shall I remove it?